core/runtime.rs

```diff
@@ -548,9 +558,9 @@ impl JsRuntime {
     isolate.set_host_initialize_import_meta_object_callback(
       bindings::host_initialize_import_meta_object_callback,
     );
-    isolate.set_host_import_module_dynamically_callback(
-      bindings::host_import_module_dynamically_callback,
-    );
+    // isolate.set_host_import_module_dynamically_callback(
+    //   bindings::host_import_module_dynamically_callback,
+    // );
```

Comment:
Is there any reason for commenting this out?

Reply:
It's by mistake. I'll revert it